tests/unit/utils/test_version.py

```diff
@@ -0,0 +1,21 @@
+from ensurepip import version
+import requests
+import pytest
+
+from csvcubed.utils.version import _get_csvcubed_version
+
+
+def test_get_csvcubed_version():
+    # The version number returned can have .dev0 appended which doesn't technically exist
+    # as a valid release url.
+    # So we check the prefix to this which should exist or if no .dev0 the whole URI is
+    # checked.
+
+    versionNumber = _get_csvcubed_version()
+
+    if versionNumber[-10:] == "0.1.0.dev0":
```

Comment:
I don't think we actually want to test that the URI exists, just that the function generates the expected.